cmd/eventkitd-sink-bigquery/main.go

```diff
@@ -0,0 +1,207 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"net"
+	"os"
+	"os/signal"
+	"runtime"
+	"syscall"
+	"time"
+
+	"cloud.google.com/go/bigquery"
+	"github.com/jtolio/eventkit/pb"
+	"github.com/jtolio/eventkit/transport"
+	"go.uber.org/zap"
+	"golang.org/x/sync/errgroup"
+)
+
+type Application struct {
+	Name     string
+	Instance string
+	Version  string
+}
+
+type Record struct {
+	Application Application
+
+	StartTime  time.Time
+	ReceivedAt time.Time
+	SendOffset time.Duration
```

Comment:
you should check out https://github.com/jtolio/eventkit/blob/main/eventkitd/main.go#L23-L50. in my view, a record == a single event, as seen by the receiver. i don't expect it to have start time, received at, or send offset. instead i would expect it to have an event timestamp and (if anything else) a timestamp correction delta. in addition to the lines above, also check out the record message definition in packet.proto